src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -22,6 +19,7 @@ class StartPolicyInterface(PolicyInterface):
     """Interface for start policies"""
 
     def __init__(self, **args):
+        self.rucio = args.pop("rucioObject", None)
```

Comment:
I would suggest to use `args.get()` here instead.

Reply:
We cannot! When we call the super class - in the line below - it has a call to `deepcopy(args)`, and if we have a class object, the whole thing blows up.
However, since you made this comment, it's not clear to me that this code deserves such comment.

Reply:
ok